Makefile.in

```diff
@@ -142,12 +155,15 @@ FIXPATHSCMD	= $(SED) $(PATHSUBS)
 FIXALGORITHMSCMD= $(SHELL) $(srcdir)/fixalgorithms $(SED) \
 		     @UNSUPPORTED_ALGORITHMS@
 
-all: $(CONFIGFILES) $(MANPAGES) $(TARGETS)
+#all: $(CONFIGFILES) $(MANPAGES) $(TARGETS)   FIXME
+all: $(IRON_TARGETS)
```

Comment:
Do you need an issue to track outstanding todos?  There's a fixme here. Looks like man pages and config files aren't being built. Not sure what else. Not sure if it matters. But the commented line should be deleted.